phovea/generator-phovea
# Check generator version on start up

Closes phovea/generator-phovea#194


I have added the `notifier.notify()` to the places where it is ensured that the message will  be printed
just once .

Since the generators compose with each other the message would be run multiple times .

![update-notfier](https://user-images.githubusercontent.com/51322092/67927268-73fc9c80-fbb8-11e9-8820-a4ee9d1c98d9.png)

I have tested every generator/subgenerator.
 
To test 

- lower the version of the generator-phovea on your local package.json,
- set `updateCheckInterval: 0` in *generators/_version/index.js* (see https://github.com/phovea/generator-phovea/pull/204/files#diff-9b7c1925207c623ed7d55e67b7af2827R15)
- run` yo phovea`
